packages/zoe/src/zoe.js

```diff
@@ -1,9 +1,9 @@
 // @ts-check
-import rawHarden from '@agoric/harden';
-import { E } from '@agoric/eventual-send';
+import harden from '@agoric/harden';
```

Comment:
I may have made a mistake here (see earlier reply). But this one isn't a merge error. I reverted these because I thought they were no longer necessary.

I point this out because we do need to be vigilant for merge errors.

Reply:
That seems outside the scope of the PR - I'm not seeing the connection between that and this PR.

Reply:
Resolving as a dup of above conversation